attributioncode/validator.go

```diff
@@ -38,6 +38,7 @@ var validAttributionKeys = map[string]bool{
 	"visit_id":       true, // https://bugzilla.mozilla.org/show_bug.cgi?id=1677497
 	"session_id":     true, // https://bugzilla.mozilla.org/show_bug.cgi?id=1809120
 	"client_id":      true, // Alias of `visit_id`.
+	"dlsource":       true, // https://github.com/mozilla-services/stubattribution/issues/159
```

Comment:
Nit: I don't think we need to add the issue here, the commit will have the reference to the PR/issue.